shell/components/form/InputWithSelect.vue

```diff
@@ -75,6 +75,14 @@ export default {
       type:    String,
       default: '',
     },
+    textRules: {
```

Comment:
We should have just one set of rules, regardless of the adopted type.

Reply:
I don't know if I agree with that. For some of the more complex sub-forms it makes sense to pass in rulesets as a subselection of the parent form's rule sets and then parse it to route the correct rulesets to the correct fields and there's methods in the mixin to do that when required but for something simple like this one that seems like overkill, especially given the amount of extra logic required to do so (given that this sub-form in particular is essentially just a mashup of two fields with some styling). In some cases, that's probably the right move but here I don't think the juice is worth the squeeze.